tests/testapp/models.py

```diff
@@ -7,3 +7,29 @@
 class Author(models.Model):
     name = models.CharField(max_length=32, unique=True)
     age = models.IntegerField()
+
+
+class Book(models.Model):
+    title = models.CharField(max_length=128)
+    author = models.ForeignKey(Author)
+
+
+class Contract(models.Model):
+    amount = models.IntegerField()
+    author = models.ForeignKey(Author)
+
+
+class Edition(models.Model):
+    name = models.CharField(max_length=32)
+    authors = models.ManyToManyField(Author)
+
+
+class Award(models.Model):
+    name = models.CharField(max_length=32)
+    authors = models.ManyToManyField(Author)
+
+
+class TypeWriter(models.Model):
+
+    model = models.CharField(max_length=32)
+    authors = models.ManyToManyField(Author)
```

Comment:
do you really need all these models for a test? Surely the order would be non-deterministic with only two referring models?